src/mmw/apps/modeling/calcs.py

```diff
@@ -36,3 +39,42 @@ def animal_population(geojson):
         'name': 'animals',
         'categories': aeu_return_values
     }
+
+
+def point_source_pollution(geojson):
+    """
+    Given a GeoJSON shape, call MapShed's `point_source_discharge` method
+    to calculate point source pollution data. Returns a dictionary to append
+    to the outgoing JSON for analysis results.
+    """
+    geom = GEOSGeometry(geojson, srid=4326)
+
+    sql = '''
+          SELECT city, npdes_id, mgd, kgn_yr, kgp_yr
+          FROM ms_pointsource
+          WHERE ST_Intersects(geom, ST_SetSRID(ST_GeomFromText(%s), 4326))
+          '''
+
+    with connection.cursor() as cursor:
+        cursor.execute(sql, [geom.wkt])
+
+        if cursor.rowcount != 0:
+            point_source = namedtuple('Point_Source_Data',
```

Comment:
Rather than creating a named tuple out of query results and then translating that into a dictionary, consider directly creating a dictionary from the query results:

```python
columns = [col[0] for col in cursor.description]
point_source_results = [dict(zip(columns, row)) for row in cursor.fetchall()]
```